lib/ansiblelint/__init__.py

```diff
@@ -196,7 +196,7 @@ def listtags(self):
     @classmethod
     def create_from_directory(cls, rulesdir):
         result = cls()
-        result.rules = ansiblelint.utils.load_plugins(os.path.expanduser(rulesdir))
+        result.rules = ansiblelint.utils.load_plugins(rulesdir)
```

Comment:
What is the advantage of moving the expansion outside?

Reply:
TBH I don't remember why I did that. Though now that it's been pointed out it would make more sense to have that in the class to ensure the rules are properly expanded.